release/train_tests/benchmark/train_benchmark.py

```diff
@@ -230,6 +293,17 @@ def get_metrics(self) -> Dict[str, float]:
                 metrics["validation/local_throughput"] * num_workers
             )
 
+        # Extra time that each worker spends to restore from checkpoint,
+        # which includes downloading the checkpoint, loading the checkpoint,
+        # and skipping through batches that were already processed.
+        restoration_time = (
+            self._metrics["checkpoint/download"].get()
```

Comment:
nit: Use constants for all keys to make it safe against typos.

Reply:
I'll do this in a followup.